packages/om-ui/src/components/Card/Card.stories.tsx

```diff
@@ -30,4 +30,17 @@ export const withTitle = () => (
   </Wrapper>
 );
 
+export const withTextAlign = () => (
+  <Wrapper>
+    <Card title="Title for card" textAlign="center">
```

Comment:
AFAIK text-align is inherited from the parents unless set, so if we set it on the Card level, ALL children will have that text align, meaning we dont have to set it explicit for each child.

Reply:
@remotealex that abs makes that children will inherit property from parent but when we are going to ever add a text in card. We are going to use a Text and title and ofcourse i dont disagree with using textAlign on card level. What more semantic to me ( can be completely wrong) would be adding a center align for Text and let the card remain dumb.

Reply:
We're passing the title="" prop though, which is a controlled element. Are you saying just to pass that prop to the `Title` element rather than having the style on the card itself? We coudl do that by then how would we apply text-align to the `children`?

Reply:
@remotealex I am actually saying to have the Text component be text-aligned rather than Card telling it's content to be the way it should be aligned.
https://material-ui.com/api/card/ ---> like if you check here, the Card is dumb about the content present in it. Like have no info and control about that.
:) Just a suggestion, (or the way I am thinking Card component to be). It could be totally different way than yours. So feel free to continue as per you feel is easy and best

Reply:
Yeah, their card doesnt have a `title` prop so it's a different usecase IMO